.github/CONTRIBUTING.md

```diff
@@ -1,65 +1,68 @@
+- A short `README.md` file with a link pointing to the tutorial or content using the sample.
+  [[Template](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/.github/new-samples-templates/README.md)]
+  - There should only be one source of truth for sample instructions (i.e. don't duplicate
+    instructions in the README that are or will be made available elsewhere).
 - A GitHub Action workflow that tests the sample code. At minimum, this should
-  dry-run any container image or Terraform configs and pass without any errors.
-  [[Example](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/.github/workflows/security-ci.yml)]
+  dry-run any container images or Terraform scripts and pass without any errors.
+  [[Template](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/.github/new-samples-templates/workflow.yml)]
   - Each container image should build successfully (e.g. `docker build...`).
-  - Each Terraform config should validate successfully (e.g. `terraform validate...`).
-  - If there are any other simple smoke tests that can be performed, they should also be added here.
+  - Each Terraform script should validate successfully (e.g. `terraform validate...`).
+  - If there are any other quick tests that can be performed, they should also be added here.
 - If the sample relies on canonical image artifacts, these can be hosted officially, which requires:
   - Cloud Build configs for all container images that pushes to the `google-samples` artifact registry.
-  [[Example](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/security/wi-secrets/cloudbuild.yaml)]
+  [[Template](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/.github/new-samples-templates/cloudbuild.yaml)]
   - A Terraform resource for the above Cloud Build configs.
   [[Example](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/.github/terraform/google-cloud-build-triggers.tf#L194-L207)]
     - Note that in order for the Cloud Build configs to be applied to the
       `google-samples` project, you need to run `terraform init && terraform apply`
       while in that project (a repository admin will do this for you). [[docs](/.github/terraform/README.md)]
     - The images will be of the form `us-docker.pkg.dev/google-samples/containers/gke<image_name>:latest`
+- Packages and dependencies pinned to up-to-date versions.
+  - Note that we have automation in place to update these versions on a weekly-basis.
+- License headers on all source code and manifest files.
+  [[Example](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/.github/new-samples-templates/cloudbuild.yaml#L1-L13)]
+- Region tags surrounding any file or snippets of code that will be embeded in a tutorial.
+  [[Example](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/ai-ml/llm-multiple-gpus/llm-service.yaml#L15-L28)]
```

Comment:
*Thought:* I was tempted to suggest something like (for all links to the YAML files):
```suggestion
  [[Example](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/ai-ml/llm-multiple-gpus/llm-service.yaml#L15-L28)]
```

But I think absolute URLs makes sense here.